_includes/metadata.html

```diff
@@ -0,0 +1,142 @@
+<!-- Standard metadata -->
+{%- if site.enable_google_verification -%}
+<meta name="google-site-verification" content="{{ site.google_site_verification }}" />
+{%- endif -%}
+{%- if site.enable_bing_verification -%}
+<meta name="msvalidate.01" content="{{ site.bing_site_verification }}" />
+{%- endif -%}
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<title>
+{%- if site.title == "blank" -%}
+{%- capture title -%}{{ site.first_name }} {{ site.middle_name }} {{ site.last_name }}{%- endcapture -%}
+{%- else -%}
+{%- capture title -%}{{ site.title }}{%- endcapture -%}
+{%- endif -%}
+{%- if page.title != "blank" and page.url != "/" -%}
+{{ title }} | {{ page.title }}
+{%- else -%}
+{{ title }}
+{%- endif -%}
+</title>
+<meta name="author" content="{{ site.first_name }} {{ site.middle_name }} {{ site.last_name }}" />
+<meta name="description" content="{%- if page.description -%}{{ page.description }}{%- else -%}{{ site.description }}{%- endif -%}" />
+{% if page.keywords or site.keywords %}<meta name="keywords" content="{%- if page.keywords -%}{{ page.keywords }}{%- else -%}{{ site.keywords }}{%- endif -%}" />{% endif %}
+<!-- OpenGraph -->
+{%- if site.serve_og_meta -%}
+<meta property="og:site_name" content="{{ site.title }}" />
+<meta property="og:type" content="website" />
+<meta property="og:title" content="{%- if page.title -%}{{ site.title }} | {{ page.title }}{%- else -%}{{ site.title }}{%- endif -%}" />
+<meta property="og:url" content="{{ page.url | prepend: site.baseurl | prepend: site.url | remove_first: 'index.html' }}" />
+<meta property="og:description" content="{%- if page.description -%}{{ page.description }}{%- else -%}{{ site.description }}{%- endif -%}" />
+{% if page.og_image or site.og_image %}<meta property="og:image" content="{%- if page.og_image -%}{{ page.og_image }}{%- else -%}{{ site.og_image }}{%- endif -%}" />{% endif %}
+<meta property="og:locale" content="{{ site.lang }}" />
+<!-- Twitter card -->
+<meta name="twitter:card" content="summary" />
+<meta name="twitter:title" content="{%- if page.title -%}{{ page.title }}{%- else -%}{{ site.title }}{%- endif -%}" />
+<meta name="twitter:description" content="{%- if page.description -%}{{ page.description }}{%- else -%}{{ site.description }}{%- endif -%}" />
+{% if page.og_image or site.og_image %}<meta name="twitter:image" content="{%- if page.og_image -%}{{ page.og_image }}{%- else -%}{{ site.og_image }}{%- endif -%}" />{% endif %}
+{%- if site.twitter_username -%}
+<meta name="twitter:site" content="@{{ site.twitter_username }}" />
+<meta name="twitter:creator" content="@{{ site.twitter_username }}" />
+{%- endif -%}
+{%- endif -%}
+<!-- Schema.org -->
+{%- if site.serve_schema_org -%}
+	<!-- Social links generator for "sameAs schema" -->
+	{%- assign sameaslinks = "" | split: "," -%}
```

Comment:
the code below looks pretty verbose... we can definitely leave this to a later PR, but this would probably work better if we nest all these profile ids in `_config.yml` under `profiles:` and then use a for loop here iterate over all of them. what do you think?

Reply:
Good idea. But in _config.yml, to keep it simple, just enter the username or id, a script that builds the urls would be necessary.
Have you thought of something specific?

It only occurs to me, a script would be made to generate all the social urls, and use that script for both the schema and the social icon bar. See if someone else comes up with another idea.

Reply:
yes, you are totally right that it would still require case-by-case URL generation... never mind  then, let's keep it as is.